src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -55,9 +56,6 @@
     <Reference Include="..\..\..\..\..\packages\System.Reflection.Metadata.$(SystemReflectionMetadataVersion)\lib\portable-net45+win8\System.Reflection.Metadata.dll" />
     <Reference Include="..\..\..\..\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll" />
     <Reference Include="..\..\..\..\..\packages\Microsoft.DiaSymReader.1.0.5\lib\net45\Microsoft.DiaSymReader.dll" />
-    <Reference Include="xunit">
-      <HintPath>..\..\..\..\..\packages\xunit.1.9.2\lib\net20\xunit.dll</HintPath>
-    </Reference>
```

Comment:
@KevinH-MS 

Also once we switch to project.json / NuGet v3 this will cease to be an issue.  The dependencies will just be stated in the project.json file and the build system / compiler will resolve them at build time.  No need to edit the project file to hard code paths.